nncf/quantization/algo.py

```diff
@@ -275,10 +277,12 @@ def get_potential_quantized_modules(self, target_model: NNCFNetwork) -> List[Pot
                 graph_operation = associated_ops[0]
                 metatype = graph_operation[InsertionPointGraph.OPERATOR_METATYPE_NODE_ATTR]
                 qconfig_list = meta_vs_qconfig_map[metatype]
-                if qconfig_list is None:
+                if qconfig_list is not None and len(qconfig_list) == 0:  # Empty list = wildcard quantization
                     qconfig_list = default_qconfig_list
-            quantized_modules_with_potential_qconfig.append(PotentialQuantizedModule(module, module_scope,
-                                                                                     qconfig_list))
+
+            if qconfig_list is not None:
+                quantized_modules_with_potential_qconfig.append(PotentialQuantizedModule(module, module_scope,
```

Comment:
`PotentialQuantizedModule` does not look self-explainable. Looking at the code I would call it `QuantizableModules`

Reply:
@asenina